przelewanka.ml

```diff
@@ -0,0 +1,73 @@
+(* Autor: Antoni Żewierżejew *)
+(* Reviewer: Jakub Nowak     *)
+
+(* zwraca NWD liczb a i b *)
```

Comment:
Powinieneś napisać w komentarzu, że `nwd 0 0 = 0` i `nwd 0 a = a`, bo z tego korzystasz, a nie jest to oczywiste.

Reply:
nwd 0 a = a jest zgodne z definicjami chociażby na wikipedii

Reply:
Ale wciąż wydaje mi się, że nie jest to oczywiste i nie szkodzi umieścić w komentarzu.
I `nwd 0 0` zgodnie z definicją nie istnieje.